### PR TITLE
adding character size to config file

### PR DESCRIPTION
**Pull request by martinacantaro**
Adds one config option to the Characters.json which is "BodySize": the size of the heatmap of a given characters, used for things like knowing if a shot hits or not

### CODE REVIEW

**Comment by martinacantaro** [comment]
closes #435

File: client/Assets/StreamingAssets/Characters.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
       "SkillActive1": "Barrel Roll",
       "SkillActive2": "Rage",
       "SkillDash": "Leap",
-      "SkillUltimate": "Fiery Rampage"
+      "SkillUltimate": "Fiery Rampage",
+      "BodySize": "10.0"
```
Comment (AminArria) [inline client/Assets/StreamingAssets/Characters.json:15]: The [docs about json fields](https://github.com/lambdaclass/curse_of_myrra/blob/main/docs/src/data_fileguides.md) have been merged please update them with new one

Comment (martinacantaro) [reply client/Assets/StreamingAssets/Characters.json:15]: Done!

**Review by tvillegas98 — APPROVED** [review]
LGTM!

**Pull request merged**